common/defs-flags.go

```diff
@@ -136,6 +136,21 @@ const (
 	FlagTaskQueueName           = "Name of the Task Queue."
 	FlagTaskQueueTypeDefinition = "Task Queue type [workflow|activity]"
 
+	// Namespace update flags
+	FlagActiveClusterDefinition           = "Active cluster name."
```

Comment:
I prefer no periods in descriptions unless it's multiple sentences. Is there prior discussion on this? Looks like the built-in options have no period:

<img width="1179" alt="image" src="https://github.com/temporalio/cli/assets/251288/f4e0b5f4-7c94-4ef8-b884-c5fadeab005c">

Reply:
i believe we added the dots because this is used for docs generation. We should be able to still remove the dots (or add when generating the docs) programmatically